library/manageiq.py

```diff
@@ -97,9 +97,9 @@ def update_provider(self, provider_id, provider_name, endpoints):
         """ Updates the existing provider with new parameters
         """
         try:
-            result = self.client.post(self.providers_url + '/%d' % provider_id,
-                                      action='edit',
-                                      connection_configurations=endpoints)
+            self.client.post(self.providers_url + '/%d' % provider_id,
```

Comment:
@dkorn not for this PR but in general you would do something like: `'%s/%d' % (self.providers_url, provider_id)` or even better (python 3 style): `"{0}/{1}".format(self.providers_url, provider_id)`